pyro/infer/autoguide/guides.py

```diff
@@ -501,22 +514,31 @@ class AutoDiagonalNormal(AutoContinuous):
         svi = SVI(model, guide, ...)
 
     By default the mean vector is initialized to zero and the scale is
-    initialized to the identity.  To change this default behavior the user
-    should call :func:`pyro.param` before beginning inference, e.g.::
+    initialized to the identity.
 
-        latent_dim = 10
-        pyro.param("auto_loc", torch.randn(latent_dim))
-        pyro.param("auto_scale", torch.ones(latent_dim),
-                   constraint=constraints.positive)
+    :param callable model: A generative model.
+    :param callable init_loc_fn: A per-site initialization function.
+        See :ref:`autoguide-initialization` section for available functions.
+    :param float init_scale: Approximate initial scale for the standard
```

Comment:
shouldn't the word 'approximate' be removed here?

Reply:
I have removed 'approximate' from `AutoNormal` and `AutoMultivariateNormal`. In the `AutoLowRankMultivariateNormal`, `init_scale` determines the rms standard deviation, since the `cov_factor` must be initialized randomly.